portfolio/src/main/webapp/script.js

```diff
@@ -80,21 +79,60 @@ async function toggleCommentHistorySection() {
 }
 
 /**
- * Fetches and adds a history of comments, and the theoretical maximum and
+ * Comments include (1) comment submission; (2) comment history.
+ * * Authenticates the current user.
```

Comment:
nit. I'm kind of confused why you're adding two '*' in some of these comments? Are you trying to tie them to (1) and (2)? If so, that should be more clear. If not, get rid of the extra '*'

Reply:
Yeah, I was trying to creating a list. I'll make this clearer